_tools/cite-guide.md

```diff
@@ -0,0 +1,459 @@
+---
+layout: default
+title: CITE Participation Guide
+collection: tools
+permalink: /tools/citeguide/
+---
+
+# Community Interoperability Test Environment (CITE) Participation Guidance

```

Comment:
Remove this header - in the templates, the title is displayed first and having an additional top level header creates **two** headers for the page

Reply:
Done